dom-events.tar/dom-events/buttons.js

```diff
@@ -0,0 +1,24 @@
+function handleClick(event){
+  console.log('button clicked');
+  console.log(event);
+  console.log(event.target);
+  document.querySelector('.click-button')
+  if (!$clickButton) throw new Error('The $clickBtn query failed');
+  element.addEventListener('click',handleClick());
+}
+function handleMouseOver(event){
+  console.log('button hovered');
+  console.log(event);
+  console.log(event.target);
+  document.querySelector('.hover-button');
+  if (!$hoverButton) throw new Error('The $hoverButton query failed');
+  element.addEventListener('mouseover',event())
+}
+function handleDoubleClick(event){
+  console.log('button doubled-clicked');
+  console.log(event);
+  console.log(event.target);
+  document.querySelector(.'double-click-button');
+  if (!$dblClickButton) throw new Error('The $dblClickButton query failed');
+  element.addEventListener('dblclick', handleDoubleClick());
```

Comment:
Again, move these outside of the function definiton, assign the result of the `querySelector` method call to a new variable (with a unique name, something like `doubleClickButton`)

and remove the `()` from the argument on line 23.